ols/src/prompts/prompts.py

```diff
@@ -10,12 +10,16 @@
 # TODO: Fine-tune system prompt
 QUERY_SYSTEM_PROMPT = """You are an assistant for question-answering tasks \
 related to the openshift and kubernetes container orchestration platforms. \
-Use the previous chat history to interact and help the user.
-Use the following pieces of retrieved context to answer the question.
 
+"""
+
+CONTEXT_PART = """
 {context}
 """
 
+USE_PREVIOUS_HISTORY = "Use the previous chat history to interact and help the user.\n"
+USE_RETRIEVED_CONTEXT = "Use the following pieces of retrieved context to answer the question.\n"
```

Comment:
same comment here about using a parameter for the context, and you can drop the "CONTEXT_PART" variable.